### ISSUE TITLE
CubicChunks Support

### ISSUE THREAD

**Issue by spacebuilder2020** [GregTechCE/GregTech#464]
Are there any plans for adding support for Cubic Chunks.  Specifically to support the ICubicPopulator and generate ores on a cube by cube basis?

Based on, #31 it supposedly works, however when testing with the latest version of Cubic Chunks it does not.

**Comment by Archengius** [comment]
yeah, there are plans, however there will be some issues regarding height-based generation - height limits don't work properly with infinite height.

**Comment by spacebuilder2020** [comment]
Yea, height based rendering is a small issue but since that is configurable, it could be configured to spawn differently in different worlds.  I think the main issue right now is to get them to spawn at all without having to hack them in.

**Comment by spacebuilder2020** [comment]
@Archengius Has there been any updates in adding some kind of support for cubic chunks.   I don't really care if the height limits don't reach infinity or not...  that can be configured in the configs.  The issue is that they don't spawn at all.
